builder/proxmox/common/config.go

```diff
@@ -287,6 +287,12 @@ func (c *Config) Prepare(upper interface{}, raws ...interface{}) ([]string, []st
 				}
 			}
 		}
+		if disk.SSD {
+			// SSD emulation is not supported on virtio device type
+			if disk.Type == "virtio" {
```

Comment:
You can make that a single directive here I would think
```suggestion
		if disk.SSD && disk.Type == "virtio" {
```

Reply:
Updated accordingly.